python/tank/platform/util.py

```diff
@@ -261,3 +261,33 @@ def get_logger(module_name):
     return logging.getLogger(full_log_path)
 
 
+def get_hyperlink_html(url, name, color=None, bold=True):
+    """
+    Provides an html string for a hyperlink pointing to the given URL
+    and displaying the provided string name.
+
+    :param str url: The URL that the hyperlink navigates to.
+    :param str name: The string name to display.
+    :param str color: The color of the text. If not given, defaults to
+                      the SG_LINK_COLOR constant.
+    :param bool bold: Whether the displayed text is bold.
+    :return: HTML string.
+    """
+    from sgtk.platform.constants import SG_STYLESHEET_CONSTANTS
+    color = color or SG_STYLESHEET_CONSTANTS["SG_LINK_COLOR"]
+    base = "<a href='{0}' style='text-decoration: none; color: {1}'>".format(
+        url,
+        color,
+    )
```

Comment:
My personal preference is to use explicit names in replacement strings rather than the `{#}` syntax. I think you could simplify here by building the link first, then adding bold at the end...

```python
url_template = "<a href='{url}' style='text-decoration: none; color: {color}'>{name}</a>".format(
    url=url,
    color=color,
    name=name
)

if bold:
    url_template = "<b>{url_template}</b>".format(url_template=url_template)
```

I *think* that would work... maybe `<b>` isn't ideal outside the `<a>`?